mscore/preferences.cpp

```diff
@@ -130,7 +130,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_EXPORT_MP3_BITRATE,                              new IntPreference(128, false)},
             {PREF_EXPORT_MUSICXML_EXPORTBREAKS,                    new EnumPreference(QVariant::fromValue(MusicxmlExportBreaks::ALL), false)},
             {PREF_EXPORT_MUSICXML_EXPORTLAYOUT,                    new BoolPreference(true, false)},
-            {PREF_EXPORT_PDF_DPI,                                  new IntPreference(300, false)},
+            {PREF_EXPORT_PDF_DPI,                                  new IntPreference(DPI, false)},
```

Comment:
Doesn’t this change the default dpi for the PDF export from 300 to 360?

If so, at least https://github.com/musescore/MuseScore/blob/master/build/Linux%2BBSD/mscore.1.in#L209 needs adjusting, and maybe the QCommandLineArg thingy used for `--help` as well.

Reply:
Thar line in the manual page is about png resolution and that has been changed to 360 a while ago already

Reply:
I wonder whether changing PDF's DPI is the right way, IMHO printers usually default to 300 DPI?

Reply:
Well, pdf is a **vectorial** format, so dpi does not really make sense for it (except if it is rasterizing things as images).
By using pdf dpi equal to internal DPI, we ensure that the matrix elements are 1 and therefore no problems with fonts should appear.

Reply:
Ah, yes, the DPI is only used for embedded images I guess.